.github/workflows/pull_request.yml

```
@@ -68,4 +68,3 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           parallel-finished: true
```

I will suggest re-adding the line, to avoid touching this file (to reduce noise) when you squash the commits.

```suggestion
          parallel-finished: true

```